deploy_production.sh

```
@@ -0,0 +1,177 @@
+#!/bin/bash
+# Production deployment script for LangGraph Cloud Webhook Handler
+# Using LangChain Smith API
+
+echo "🚀 LangGraph Cloud Webhook Handler - Production Deploy"
+echo "======================================================"
+
+# Load environment variables from .env.webhook
+if [ -f ".env.webhook" ]; then
+    export $(cat .env.webhook | grep -v '^#' | xargs)
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)